packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -285,32 +287,63 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({
             <ComponentPalette />
           </Box>
         </div>
-        <ResizablePanes
-          leftPane={
-            <QueryEditor
-              iotSiteWiseClient={iotSiteWiseClient}
-              iotTwinMakerClient={iotTwinMakerClient}
-            />
-          }
-          centerPane={
-            <div
-              className='display-area'
-              ref={(el) => setViewFrameElement(el || undefined)}
-              style={{ backgroundColor: colorBackgroundCellShaded }}
-            >
-              <GestureableGrid {...gridProps}>
-                <ContextMenu {...contextMenuProps} />
-                <Widgets {...widgetsProps} />
-                {!widgetLength && <DashboardEmptyState />}
-                {activeGesture === 'select' && (
-                  <UserSelection {...selectionProps} />
-                )}
-              </GestureableGrid>
-              <WebglContext viewFrame={viewFrame} />
-            </div>
-          }
-          rightPane={propertiesPanel}
-        />
+        {showPlayground() ? (
```

Comment:
can we split this conditional into its own components instead of doing one large return? that way its easier to debug

Reply:
Sure throwing the creator into a component should help with readability